packages/cli/src/commands/init/init.js

```diff
@@ -79,13 +80,19 @@ export default function initialize(
      * Commander is stripping `version` from options automatically.
      * We have to use `minimist` to take that directly from `process.argv`
      */
-    const version: string = minimist(process.argv).version || 'latest';
+    const version: string = minimist(process.argv).version;
+
+    if (version && !semver.satisfies(version, '0.60.0')) {
```

Comment:
this should also check if default template from `react-native/templates` was used

Reply:
Maybe we should just move this check to `createFromReactNativeTemplate`?

Reply:
sure